### PR TITLE
inference-triton-cli

### PR DESCRIPTION
**Pull request by ArzooAneja**
# PR into Azure/azureml-examples

## Checklist

I have:

- [x] read and followed the contributing guidelines

## Changes

-

fixes #

### CODE REVIEW

File: cli/endpoints/online/triton/multi-models/multi-endpoint-aks.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    etblue: 100
+deployments:
+    - name: etblue
+      model: azureml:multi-models:2
+      code_configuration:
+          code:
+             local_path: ./code
+          scoring_script: dummy.py
+      environment:
+            name: aks-multi-env
```
Comment (partsrini) [inline cli/endpoints/online/triton/multi-models/multi-endpoint-aks.yml:15]: Hi: is there a version for MIR v2? would suggest defaulting to MIR and have a comment for aks in the same yaml file.

File: cli/endpoints/online/triton/multi-models/multi-endpoint-aks.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+            version: 1
+            docker:
+                image: metricstestacr.azurecr.io/triton-base:3
```
Comment (msjelly) [inline cli/endpoints/online/triton/multi-models/multi-endpoint-aks.yml:18]: In all yaml files, update environment.docker.image to mcr.microsoft.com/azureml/tritonserver-21.02-py38-inference:20210525.1

File: cli/endpoints/online/triton/multi-models/multi-endpoint-aks.yml (hunk 1 above)
Comment (msjelly) [inline cli/endpoints/online/triton/multi-models/multi-endpoint-aks.yml:15]: Environment depends on the image.  Same Environment is used for aks, mir.   Remove 'aks' or 'mir' from environment names as that will be misleading for the user.

Comment (ArzooAneja) [reply cli/endpoints/online/triton/multi-models/multi-endpoint-aks.yml:15]: Done as per comments

File: cli/endpoints/online/triton/multi-models/multi-endpoint-aks.yml (hunk 2 above)
Comment (ArzooAneja) [reply cli/endpoints/online/triton/multi-models/multi-endpoint-aks.yml:18]: Done as per comments

File: cli/endpoints/online/triton/multi-models/multi-endpoint-aks.yml (hunk 1 above)
Comment (ArzooAneja) [reply cli/endpoints/online/triton/multi-models/multi-endpoint-aks.yml:15]: Hi Srini,
We have updated the yml files as below -
'aks-multi-env' to 'multi-env' 
'mir-multi-env' to 'multi-env'.

Please review and let us know for any updates.

Comment (partsrini) [reply cli/endpoints/online/triton/multi-models/multi-endpoint-aks.yml]: Thanks

**Comment by msjelly** [comment]
Please do not use keyword 'lite' anywhere - e.g. file name, variable name etc.  For example, rename single-triton-lite.ipynb to single-model-triton.ipynb.

**Comment by msjelly** [comment]
In all .ipynb files, replace 'service' with 'endpoint'.

**Comment by msjelly** [comment]
For AKS online endpoint, change comment "Get Auth Token" to "Get key for online endpoint".
For Managed online endpoint, change comment "Get Auth Token" to "Get token for online endpoint."

**Comment by msjelly** [comment]
Be consistent in style for variable names and use all lower case.  e.g. Replace  Service_key with service_key.

**Comment by msjelly** [comment]
In .ipynb notebooks, replace "Test Webservice" with "Test Online Endpoint".

**Comment by msjelly** [comment]
Please remove '-checkpoint' from notebook(.ipynb) filenames.  In order to drive this PR to completion, please do not introduce changes other than to resolve comments.

**Comment by gvashishtha** [comment]
This PR needs cleaning up. Specifically:

1. Don't track changes on files you aren't changing. Right now there are ~10 .ipynb files with a newline at the end that should not be tracked in this PR. Use `git rm` to remove
2. You shouldn't need .ipynb files at all. Do your deployment _and_ your scoring in the .sh file, using `curl`

**Comment by buchananwp** [comment]
Work needed before approval: 
- [ ] fix merge conflicts
- [ ] get rid of the .ipynb files (everything done through .sh)
- [ ] ensure we have descriptive variable names
- [ ] make sure the modularization of Python code is consistent (I think I see preprocessing being done both through helper functions and through ensembles)
- [ ] get rid of unused files

**Comment by buchananwp** [comment]
Work needed before approval: @ArzooAneja 
- [ ] fix merge conflicts
- [ ] get rid of the .ipynb files (everything done through .sh)
- [ ] ensure we have descriptive variable names
- [ ] make sure the modularization of Python code is consistent (I think I see preprocessing being done both through helper functions and through ensembles)
- [ ] get rid of unused files

**Pull request closed without merging**